### ISSUE TITLE
some time skill doesn't properly show up

### ISSUE THREAD

**Issue by Orionox** [CookieLuc/skill-extensions#3]
![Capture](https://user-images.githubusercontent.com/20059838/178109601-1837b10d-7509-4951-ba61-cfef45d2a7bd.PNG)
This is only happening to a single one of my players so its a super small problem, but I figured you should know.

**Comment by RavenknightDM** [comment]
Yep, we got the same problem. But only on the two players that are proficient with the skill. I know that it used to work and look fine before.
